packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -388,7 +392,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             "connect",
             connection,
             checkpointSequenceNumber !== undefined ?
-                this.lastObservedSeqNumber - this.lastSequenceNumber : undefined);
+                this.lastObservedSeqNumber - this.lastProcessedSequenceNumber : undefined);
```

Comment:
Maybe it's worth (as separate PR) rename these, for easier reading. Ideally, we follow this schema - lastSequenceNumber  (public getter) - _lastSequenceNumber  (private)
That said, I think it's not very safe to rely on a fact here (I think that's the case) that they have vastly different meaning (virtualized vs. not) - developers who fix bugs in this code will accidently use these interchangeably, unless we have really good test coverage to catch it (essentially some test that validates public getters are not called form within this class by having derived test class that catches such things)